make the hpn patch apply when the x509 patch has also been applied

--- openssh-6.6p1-hpnssh14v4.diff
+++ openssh-6.6p1-hpnssh14v4.diff
@@ -1742,18 +1742,14 @@
  	if (options->ip_qos_interactive == -1)
  		options->ip_qos_interactive = IPTOS_LOWDELAY;
  	if (options->ip_qos_bulk == -1)
-@@ -345,9 +393,10 @@
+@@ -345,6 +393,7 @@
  	sUsePrivilegeSeparation, sAllowAgentForwarding,
  	sHostCertificate,
  	sRevokedKeys, sTrustedUserCAKeys, sAuthorizedPrincipalsFile,
-+	sTcpRcvBufPoll, sHPNDisabled, sHPNBufferSize,
++	sTcpRcvBufPoll, sHPNDisabled, sHPNBufferSize, sNoneEnabled,
  	sKexAlgorithms, sIPQoS, sVersionAddendum,
  	sAuthorizedKeysCommand, sAuthorizedKeysCommandUser,
--	sAuthenticationMethods, sHostKeyAgent,
-+	sAuthenticationMethods, sNoneEnabled, sHostKeyAgent,
- 	sDeprecated, sUnsupported
- } ServerOpCodes;
- 
+ 	sAuthenticationMethods, sHostKeyAgent,
 @@ -468,6 +517,10 @@
  	{ "revokedkeys", sRevokedKeys, SSHCFG_ALL },
  	{ "trustedusercakeys", sTrustedUserCAKeys, SSHCFG_ALL },
